helm/cluster-api-monitoring/values.yaml

```diff
@@ -6,9 +6,9 @@ project:
   commit: "[[ .SHA ]]"
 
 image:
-  registry: docker.io
+  registry: quay.io
   name: giantswarm/kube-state-metrics
-  tag: v2.7.0
+  tag: v2.7.0-b7a7070
```

Comment:
@QuentinBisson are you fine with the custom build of KSM main branch?

Once the next KSM version got released we could switch back to upstream image

Reply:
If we have no choice go for it.